decorator.rb

```diff
@@ -0,0 +1,24 @@
+require_relative './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
```

Comment:
- Hey, let's also add the `attr_accessor :nameable` for the nameable so this could be accessed.